### DISCUSSION TITLE
Getting "Failed to get document because the client is offline" with Firestore version 22.0.2

### DISCUSSION THREAD

**Discussion by fractalbob** [firebase/firebase-js-sdk#4374] [Archive] [answered]
My app had been running fine when suddenly today I started getting the above message. So I upgraded the Firestore library to version 22.0.2 and I'm still seeing it. 

I know this problem has been around a long time, but now it's gotten a lot worse. Performance accessing Firestore has degraded also since today. 
![Screenshot from 2021-02-01 06-03-27](https://user-images.githubusercontent.com/1521677/106417163-9fbda080-6453-11eb-9892-6a3d4a031004.png)
![Screenshot from 2021-02-01 06-02-20](https://user-images.githubusercontent.com/1521677/106417165-a0563700-6453-11eb-9733-30a01d603296.png)
![Screenshot from 2021-02-01 06-01-51](https://user-images.githubusercontent.com/1521677/106417166-a0563700-6453-11eb-8c4e-82e0b6391fa6.png)

**Comment by samtstern** [comment, marked as answer]
@fractalbob that is an Android issue, this repository is for the JavaScript SDK.
